package.json

```diff
@@ -50,36 +50,169 @@
     "permissionSets": [
       {
         "permissionName": "module.finance.enabled",
-        "displayName": "UI: Finance module is enabled",
-        "visible": true
```

Comment:
useless changes

Reply:
per permissions spreadsheet, it shouldn't be visible for users

Reply:
but default value is `true`, and it was already here. So it's like you get a bonus for more lines touched, but it's not the case

Reply:
set `visible` to `false`

Reply:
yeah, my bad, I was referring to https://github.com/folio-org/stripes-cli/blob/master/doc/commands.md#perm-create-command where `--visible default: true`